pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["setuptools>=42"]
```

Comment:
do we need that >=42 constraint for setuptools?

Reply:
I copied that from the SDK repo. Wasn't sure if it was needed so kept it.